src/equations/compressible_euler_multicomponent_2d.jl

```diff
@@ -1,67 +1,96 @@
 
 @doc raw"""
-    CompressibleEulerMulticomponentEquations2D
+    CompressibleEulerMulticomponentEquations2D(gammas, gas_constants)
 
 !!! warning "Experimental code"
     This system of equations is experimental and can change any time.
 
-Multicomponent version of the compressible Euler equations for an ideal gas in two space dimensions.
+Multicomponent version of the compressible Euler equations 
+```math
+\begin{equation*}
+\partial t
+\begin{pmatrix}
+\rho v_1 \\ \rho v_2 \\ E \\ \rho_1 \\ \rho_2 \\ \vdots \\ \rho_{n}
+\end{pmatrix}
++
+\partial x
+\begin{pmatrix}
+\rho v_1^2 + p \\ \rho v_1 v_2 \\ (E+p) v_1 \\ \rho_1 v_1 \\ \rho_2 v_1 \\ \vdots \\ \rho_{n} v_1
+\end{pmatrix}
++
+\partial y
+\begin{pmatrix}
+\rho v_1 v_2 \\ \rho v_2^2 + p \\ (E+p) v_2 \\ \rho_1 v_2 \\ \rho_2 v_2 \\ \vdots \\ \rho_{n} v_2
+\end{pmatrix}
+= 
+\begin{pmatrix}
+0 \\ 0 \\ 0 \\ 0 \\ 0 \\ \vdots \\ 0
+\end{pmatrix}
+\end{equation*}
+```
+for calorically perfect gas in two space dimensions.
+
+The specific heat ratios 'gammas' and the gas constants 'gas_constants' in [kJ/(kg*K)]
+should be passed as tuples e.g. gammas=(1.4, 1.667) or gas_constants=(0.4,) in the one component case.
+
+The remaining variables like the specific heats at constant volume 'cv' or the specific heats at 
+constant pressure 'cp' are then calculated considering a calorically perfect gas.
 """
-struct CompressibleEulerMulticomponentEquations2D{RealT<:Real} <: AbstractCompressibleEulerMulticomponentEquations{2, 5}
-  gamma1          ::RealT 
-  gamma2          ::RealT
-  gas_constant1   ::RealT
-  gas_constant2   ::RealT
-  cv1             ::RealT
-  cv2             ::RealT
-  cp1             ::RealT
-  cp2             ::RealT
-end
 
-function CompressibleEulerMulticomponentEquations2D()
+struct CompressibleEulerMulticomponentEquations2D{NCOMP, NVARS, RealT<:Real} <: AbstractCompressibleEulerMulticomponentEquations{2, NVARS}
+  gammas            ::SVector{NCOMP, RealT} 
+  gas_constants     ::SVector{NCOMP, RealT}
+  cv                ::SVector{NCOMP, RealT}
+  cp                ::SVector{NCOMP, RealT}
 
-  # Set ratio of specific heat of the gas per species 
-  # (It holds: gamma = cp/cv) (Can be used for consistency check!)
-  gamma1  = 1.4
-  gamma2  = 1.4
+  function CompressibleEulerMulticomponentEquations2D{NCOMP, NVARS, RealT}(gammas, gas_constants) where {NCOMP, NVARS, RealT<:Real} 
```

Comment:
For convenience, you could add an outer constructor for
```julia
  CompressibleEulerMulticomponentEquations2D(gammas::Real, gas_constants::Real) = CompressibleEulerMulticomponentEquations2D(tuple(gammas), tuple(gas_constants))
```
In that case, you can also remove the one component example from the docstring.

Reply:
I checked 

CompressibleEulerMulticomponentEquations2D(gammas          = 1.4,
                                                                                  gas_constants = 0.4)
                                                                                  
 as well as
 
 CompressibleEulerMulticomponentEquations2D(gammas          = (1.4),
                                                                                  gas_constants = (0.4))
                                                                                  
and it looks like this additional ctor wouldn't even be necessary.

Reply:
Please note that `(1.4)` is identical to `1.4`, i.e., they are both of type `Float64`. To get a tuple with one component, you need to provide a comma for the parser to distinguish it from regular parens, i.e., `(1.4,)`.

If the scalar versions work, I suggest to modify the elixirs such that they don't use tuples in this case.

Reply:
done in c7e9a75